### PR TITLE
New App: Weather Dashboard for displaying weather info from an Ambient Weather station

### PR DESCRIPTION
**Pull request by ckelley-1992**
This app pulls from an Ambient Weather station to display temperature, wind, and precipitation for both rain event and weekly totals.

### CODE REVIEW

**Comment by ckelley-1992** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ckelley-1992** [comment]
recheck

**Comment by ckelley-1992** [comment]
weather_dashboard

File: apps/weatherdashboard/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: weather-dashboard
+name: Weather Dashboard
```
Comment (matslina) [inline apps/weatherdashboard/manifest.yaml:3]: Would be great if the metadata could be updated to reflect that it's specifically weather stations from Ambient Weather that's supported.

File: apps/weatherdashboard/weather_dashboard.star [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                icon = "brush",
+                default = "#1167B1",
+            ),
+        ],
+    )
+
+# pragma mark Helper Methods
+
+def get_weather_data(config):
+    cached_data = cache.get("cached_data")
```
Comment (matslina) [inline apps/weatherdashboard/weather_dashboard.star:79]: Cache is scoped to the app, but shared among users. This cache key will need to incorporate something unique to the user, such as their API and application keys.

Comment (ckelley-1992) [reply apps/weatherdashboard/weather_dashboard.star]: Ah I didn't realize that, just updated it to use the api key to disambiguate

File: apps/weatherdashboard/manifest.yaml (hunk 1 above)
Comment (ckelley-1992) [reply apps/weatherdashboard/manifest.yaml:3]: I updated the description to reference Ambient Weather specifically. I'm running into character limits on the name but will be happy to change it if you have anything specific

**Review by matslina — APPROVED** [review]

**Pull request merged**